node/src/Commands.ts

```diff
@@ -3459,3 +3459,75 @@ export function createBZPopMin(
 ): command_request.Command {
     return createCommand(RequestType.BZPopMin, [...keys, timeout.toString()]);
 }
+
+/**
+ * Optional arguments to getex command.
+ */
+export type GetExOptions = {
+    expiry: /**
+     * Retain the time to live associated with the key. Equivalent to
+     * `PERSIST` in the VALKEY API.
+     */
+    | "persist"
+        | {
+              type: /**
+               * Set the specified expire time, in seconds. Equivalent to
+               * `EX` in the VALKEY API.
+               */
+              | "seconds"
```

Comment:
Maybe use an enum which is backed by corresponding strings?
```
enum blah {
  seconds = 'ex'
  ...
}
```

Reply:
This type actually follows what set has for its options. The point is that for measurements other than "persist" which should be passed as is, all the other ones should be followed by a number count, hence the type structure. HWhy does it look broken?